core/README.md

```diff
@@ -159,6 +159,8 @@ The examples are available at [here](../examples/rust).
 
 Check out the [CONTRIBUTING](CONTRIBUTING.md) guide for more details on getting started with contributing to this project.
 
-## License
+## Trademarks & Licenses
```

Comment:
Let's use "License and Trademarks" everywhere.

Reply:
done!